content/applications/inventory_and_mrp/quality/quality_management/failure_locations.rst

```diff
@@ -0,0 +1,107 @@
+=================
+Failure locations
+=================
+
+.. |QCP| replace:: :abbr:`QCP (Quality Control Point)`
+.. |QCPs| replace:: :abbr:`QCP (Quality Control Points)`
+
+In Odoo, quality control points (QCPs) are used to create quality checks, which prompt employees to
+confirm the quality of products, when they are included in certain operations. By setting one or
+more *failure locations* on a |QCP|, products that fail the quality checks it creates can be sent to
+one of the specified locations.
```

Comment:
```suggestion
In Odoo, *Quality Control Points (QCPs)* are used to create quality checks, which prompt employees
to confirm the quality of products, when they are included in certain operations.

By setting one or more *failure locations* on a |QCP|, products that fail the quality checks can be
sent to a specified location.
```
suggesting the use of italics here because this is, essentially, briefly defining QCPs